multisrc/overrides/madara/randomscan/src/luratoon.kt

```diff
@@ -7,9 +7,9 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class RandomScan : Madara(
+class luratoon : Madara(
     "Random Scan",
-    "https://randomscanlators.net",
+    "https://luratoon.com/",
```

Comment:
```suggestion
    "https://luratoon.com",
```
remove trailing slash